api/schema.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS "benchmark"."history" (
```

Comment:
isn't it possible to use it from cr8? So we don't have to update it here if a new cr8 version brings a new schema